spec/custom_matchers.rb

```diff
@@ -0,0 +1,12 @@
+RSpec::Matchers.define :belong_to do |expected|
+  match { |actual| (actual.to_a.uniq - expected.to_a.uniq).empty? }
+end
```

Comment:
Basically, Redis doesn't necessarily insert keys in the given order. Hence, this is used for checking when specific keys / pattern is given.

This matcher is close to, but not exactly "same arrays ignoring order". Rather, it's intended to check if `actual` is a "subset array" of `expected`.  (Inverted `include` matcher?)